scripts/validate-copyright-header.sh

```diff
@@ -9,7 +9,7 @@ echo "==> Checking copyright headers <=="
 
 files=$(find . -type f -iname '*.go' ! -path './vendor/*' ! -path './hack/tools/*' ! -path './test/e2e/vendor/*')
 licRes=$(for file in $files; do
-           awk 'NR<=3' "$file" | grep -Eq "(Copyright|generated|GENERATED)" || echo "$file";
+           awk 'NR<=4' "$file" | grep -Eq "(Copyright|generated|GENERATED)" || echo "$file";
```

Comment:
what is this change doing?

Reply:
golang v1.17 (probably more like gofmt) brings a new canonical build tags format that includes a newline, so there are a few files that have one more line above the grep match pattern than previously.

If I were testing your code review skills you'd get an A+, good job!